publishing/SiteObject.py

```diff
@@ -0,0 +1,197 @@
+'''Classes for files published to S3'''
+
+import mimetypes
+import hashlib
+import gzip
+import binascii
```

Comment:
i tend to alphabetize imports within these groups for readability; this isn't a pep8 rule, simply a personal pref (no change necessary here)